analysis_options.yaml

```diff
@@ -93,20 +98,25 @@ linter:
     - prefer_void_to_null
     - provide_deprecation_message
     - recursive_getters
+    - require_trailing_commas
     - slash_for_doc_comments
     - sort_pub_dependencies
+    - sort_unnamed_constructors_first
```

Comment:
I guess?
I tend to not like the strict member ordering type lints... the 2 examples seem like a worthwhile change though? 🤷